README.md

```diff
@@ -1439,6 +1439,7 @@ Table of Contents
    * [wakatime.com](https://wakatime.com/) — Quantified self-metrics about your coding activity, using text editor plugins, limited plan for free.
    * [WebComponents.dev](https://webcomponents.dev/) — In-browser IDE to code web components in isolation with 58 templates available, supporting stories and tests.
    * [PHPSandbox](https://phpsandbox.io/) — Online developement environment for PHP.
+   * [Google Collab](https://colab.research.google.com/) - Free Jupyter Notebooks development environment. 
```

Comment:
Probsbly better with the other Google stuff - however this isn’t really a SaaS it’s a research project, Google is notorious for shutting these down or radically changing them so I am reluctant to add these

Reply:
I don't think Google is ever going to shut it down because its a very successful project, it even has a [paid plan](https://colab.research.google.com/signup) with more features and there are lots of paying customers as well. It temporary provides a cloud vm  , which google already give for free in google cloud (even without entering the trial plan). And it uses the google drive's storage. The only difference is that google cloud gives raw terminal, while this is a hosted jupyter notebook service. And it has made learning and developing machine learning programs very easy. It provides high perormance GPUs & TPUs for machine learning for free. I don't think its gonna shut anytime soon. 

And regarding to keep with with other google cloud stuff, I don't think this would fall in the same category, because this isn't for hosting stuff on the cloud, its mainly for running you python programs or training you machine learning models. So just running python scripts won't make it a cloud provider.

Reply:
Sure, but also https://killedbygoogle.com/

Reply:
Ya, ik but I personally don't think its gonna shutdown anytime soon. It started in 2017 and has been really successful since

Reply:
Google FaaS, Vision AI, Speech-to-Text, NLP API, AutoML Tables, etcetc all fall in the same bucket - all would be listed under googles free services - and are all listed in one place at https://cloud.google.com/free

please add to existing secton